ffv1.md

```diff
@@ -672,6 +672,8 @@ ESC MUST NOT be used if the value can be coded as non ESC.
 
 ##### Examples
 
+Below are pratical examples of how Signed Golomb Rice Codes are decoded based on the serie of bits extracted from the bitstream as described by the method above:
```

Comment:
I suggest (fixes 2 spelling errors and adds a table cross-reference
> [@tableGolombExamples] shows practical examples of how Signed Golomb Rice Codes are decoded based on the series of bits extracted from the bitstream as described by the method above:

And add this line after the table:
> Table: Examples of decoded Signed Golomb Rice Codes. {#tableGolombExamples}

Reply:
Commit updated, @michaelni please review/merge.